spec/services/prison_visits/api_spec.rb

```diff
@@ -135,10 +135,6 @@
   describe 'request_visit', vcr: { cassette_name: 'request_visit' } do
     subject { super().request_visit(valid_booking_params) }
 
-    before do
```

Comment:
You're right there would be tests on the PVB2 side for that, but what I've been doing is these tests is a bit more than just asserting that the correct requests are made. It's closer towards a contract test, which tests that from this app's perspective the API is behaving as claimed – I'm reading this retrospectively, but it describes a very similar approach: http://martinfowler.com/bliki/IntegrationContractTest.html.